components/builder-web/app/origin/origin-page/origin-integrations-tab/origin-integrations-tab.component.html

```diff
@@ -2,27 +2,89 @@
   <div>
     <h3>
       <hab-icon symbol="docker"></hab-icon>
-      Docker Hub Integration
+      Container Registries
     </h3>
     <p>
-      By adding this integration, the Habitat Build Service can export the result of your package build jobs (.hart file) to a
-      Docker container and publish it to your Docker Hub account.
+      When connecting a plan file to the Habitat Build Service, you can optionally export the result of your package build jobs
+      (.hart file) to a Docker container and publish it to your registry account(s).
     </p>
-    <div *ngIf="integrations.docker.size === 0">
-      <a mat-raised-button color="primary" (click)="addDocker()">Add Docker Hub account</a>
-    </div>
+    <mat-grid-list cols="2" gutterSize="15px" rowHeight=60 class="registries">
```

Comment:
Possible to set this stuff with CSS?

Reply:
those are set in the `style` tag so it's not possible to override them